lib/membrane/rtp/vad.ex

```diff
@@ -121,6 +121,8 @@ defmodule Membrane.RTP.VAD do
 
   defp handle_if_present(buffer, nil, state), do: {{:ok, buffer: {:output, buffer}}, state}
 
+  @timestamp_limit Bitwise.bsl(1, 32)
```

Comment:
```suggestion
  @timestamp_limit Bitwise.bsl(1, 32) - 1
```

And also make sure that you have applied the subtraction in each place correctly.